runner-install.sh

```diff
@@ -2,9 +2,9 @@
 
 # Installing make-profiler
 sudo apt install -y python3-pip graphviz gawk
-sudo pip3 install slack slackclient
-sudo pip3 install https://github.com/konturio/make-profiler/archive/master.zip
-sudo pip3 install pandas
+pip3 install slack slackclient
+pip3 install https://github.com/konturio/make-profiler/archive/master.zip
+pip3 install pandas
```

Comment:
probably pandas should not be installed in geocint-runner. for example it is not used in geocint-mapaction.

Reply:
yeah, will remove from runner-install.sh

Reply:
removed